style.css

```diff
@@ -52,15 +52,14 @@ header {
   display: flex;
```

Comment:
- [ ] [OPTIONAL] It is noticed that `display: flex` property is repeated multiple times, Kindly let us stick to the DRY principle by not repeating a property more than it should 🙂. This can be done by having a class called `.flex-row` or `.flex-column` and assigning it wherever possible as a class name in your `html` markup code.
```
.flex-row {
  display: flex;
}

.flex-column {
  display: flex;
  flex-direction: column;
}
```
- And kindly attend to any repetitions in your code 🙂.
- Kindly note that DRY is a requirement(attracts required changes), but since previous reviewers have been overlooking it, I've decided to make it optional.